a10_octavia/tests/unit/common/test_utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020, A10 Networks.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from copy import deepcopy
+from oslo_config.cfg import ConfigFileValueError
+
+from a10_octavia.common.data_models import VThunder
+import a10_octavia.common.utils as utils
+from a10_octavia.tests.unit.base import BaseTaskTestCase
```

Comment:
Styling guide for hacking dictates [not using direct object imports](https://docs.openstack.org/hacking/latest/user/hacking.html#imports)
```python
from oslo_config.cfg import ConfigFileValueError

from a10_octavia.common.data_models import VThunder
...
from a10_octavia.tests.unit.base import BaseTaskTestCase
```

Reply:
Will do this